include/sys/cbprintf.h

```diff
@@ -49,22 +49,25 @@ extern "C" {
  * @{
  */
 
-/** @brief Required alignment of the buffer used for packaging. */
+#define POWER_OF_TWO(x) ((x) && !((x) & ((x) - 1)))
```

Comment:
Generally, local macro definitions like these should be avoided to prevent conflicts; if it is necessary, the relevant toolchain or util headers should be updated to support them.

Reply:
Done